recipe/bld.bat

```diff
@@ -23,5 +24,5 @@ nmake install
 if errorlevel 1 exit 1
 
 :: Move everything 1-level down.
-xcopy %LIBRARY_INC%\freetype2\freetype\*.* %LIBRARY_INC%\freetype2\ /s /e || exit 1
-rmdir %LIBRARY_INC%\freetype2\freetype /s /q
+move %LIBRARY_INC%\freetype2\freetype %LIBRARY_INC% || exit 1
+move %LIBRARY_INC%\freetype2\ft2build.h %LIBRARY_INC% || exit 1
```

Comment:
What is driving this behaviour in the first place? If freetype does it one particular way, should we honour that? After all, downstream builders need to deal with the weird folder choice for non conda builds, no?